src/Guide/Views.hs

```diff
@@ -42,11 +43,15 @@ import NeatInterpolation
 import Lucid hiding (for_)
 -- Network
 import Data.IP
+import Network.HTTP.Conduit
```

Comment:
Why conduit if you're only using things that `Network.HTTP.Conduit` reexports from http-client? Also, if you use `http-client-tls` instead you'll be able to query `https` links as well.